riff-raff-tools.yaml

```diff
@@ -0,0 +1,18 @@
+stacks: [deploy]
```

Comment:
Currently this riff-raff.yaml is almost identical to the CAPI version, but as we start having a version of this Step Function for different numbers of ASGs they will start to diverge...